llama_stack/apis/common/job_types.py

```diff
@@ -3,7 +3,9 @@
 #
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
+from datetime import datetime
 from enum import Enum
+from typing import Optional
```

Comment:
We should stop adding Optional imports because lately Python support this natively with `type | None` construct.

Reply:
done